Issue #8: added a new option '-b' equivalent to '--label'.

mike1813 opened the pull request:
This argument allows the rdfs:label property of the domain model to be specified, overriding the 'label' field in DomainModel.csv.

The string '-UNEXPANDED' is still appended if the domain model says it supports asset populations, but the csv2nq code is run without specifying the '-e' option so that the NQ file only supports singleton assets.

I found this necessary because during domain model development, I would specify the new version as 'domain-network-testing' so I could keep the current working version and the baseline version on one system-modeller. But I kept forgetting to change the label field in DomainModel.csv, so once deployed I couldn't tell which model was which as both the baseline and testing version were labelled as 'NETWORK'.

@kenmeacham : I asked you to do a review on this, but you may want to wait because further changes will be added on top and be the subject of further pull requests.

mike1813 commented:
My only real concern here is whether we may need to change the CI pipeline to include the new argument. If so, it isn't clear what value we would specify, so my guess is we can leave the CI pipeline alone and use the new argument only when generating an NQ file manually (as in a domain model development-test cycle).

kenmeacham commented on csv2nq.py:
```diff
@@ -119,13 +119,19 @@ def output_domain_model(nqw, unfiltered, heading):
+        if args["label"]:
+            label = args["label"]
+        else:
+            label = row[label_index]
```
This else clause seems unnecessary, as the same (default) value is set above. I would just remove these 2 lines.

mike1813 commented:
@kenmeacham : I just removed the two redundant lines and updated the README.md in branch 8, addressing your comments. Please check it is OK, and if so approve the pull request and get it merged.

Then you could move on to pull requests 15 and 16.

Ideally, if there are no issues we should get these three pull requests put to bed by the end of this week. Next week I need to talk to @samuelsenior about my other set of changes, dealing with modularity of domain models, which depends on these.
